userspace/libscap/engine/modern_bpf/scap_modern_bpf.c

```diff
@@ -98,6 +103,43 @@ static int32_t populate_64bit_interesting_syscalls_table(bool* ppm_sc_array)
 	return ret;
 }
 
+static int32_t check_minimum_kernel_version(char* last_err)
+{
+	uint32_t major = 0;
+	uint32_t minor = 0;
+	uint32_t patch = 0;
+
+	struct utsname info;
+	if(uname(&info) != 0)
+	{
+		if(last_err != NULL)
+		{
+			snprintf(last_err, SCAP_LASTERR_SIZE, "unable to get the kernel version with uname: %s", strerror(errno));
+		}
+		return SCAP_FAILURE;
+	}
+
+	if(sscanf(info.release, "%u.%u.%u", &major, &minor, &patch) != 3)
+	{
+		if(last_err != NULL)
+		{
+			snprintf(last_err, SCAP_LASTERR_SIZE, "unable to get major, minor, patch with sscanf: %s", strerror(errno));
```

Comment:
How about adding `info.release` to this error message? For ease of debugging.

Reply:
smart idea thank you :)!

Reply:
done :)